internal/tray-agent/agent/client/advertisementController.go

```diff
@@ -8,11 +8,7 @@ import (
 
 //createAdvertisementController creates a new CRDController for the Liqo Advertisement CRD.
 func createAdvertisementController(kubeconfig string) (*CRDController, error) {
-	controller := &CRDController{
-		addFunc:    advertisementAddFunc,
-		updateFunc: advertisementUpdateFunc,
-		deleteFunc: advertisementDeleteFunc,
-	}
```

Comment:
why did you remove this advertisement function?

Reply:
In this new version, the main point of access is the foreign cluster. It's the ForeignClusterController's cache that monitors with its handlers the changes on a FC, like the 'joined' field for the peerings.

I considered at least for this moment to remove the handling of advertisements events, since a notification would now be useless and there is no associated behavior to these changes.

Moreover, in the next PR I'll bring some refactoring to the Listener mechanism, and so there would be additional useless code to refactor